pylzy/lzy/api/v1/call.py

```diff
@@ -159,7 +159,6 @@ def lazy(*args, **kwargs):
             return f(*args, **kwargs)
 
         prov = active_workflow.provisioning.override(provisioning)
-        prov.validate()
```

Comment:
Why removing validation here?

Reply:
With the new logic of merging and pool inference, I think there is only one place with validation that makes sense - right before the inference itself, because there are maybe any non-valid combinations in the middle.
Also, new validation is much less intense.